src/coreclr/ilasm/grammar_before.cpp

```diff
@@ -51,6 +51,7 @@ static char* newString(_In_ __nullterminated const char* str1);
 static void corEmitInt(BinStr* buff, unsigned data);
 static void AppendStringWithLength(BinStr* pbs, _In_ __nullterminated char* sz);
 static void AppendFieldToCustomBlob(BinStr* pBlob, _In_ BinStr* pField);
+static unsigned corCountArgs(BinStr* args);
```

Comment:
Also append below lines after this line:
```cpp
extern void yyerror(_In_ __nullterminated const char*);
extern Instr* SetupInstr(unsigned short);
extern int yylex();
```

Reply:
Can you clarify why extern needed?